style.css

```
@@ -429,4 +429,11 @@ a.wp-block-social-link-anchor::after {
 
 .full-excerpt .posts-list-query .wp-block-post-excerpt__excerpt {
   -webkit-line-clamp: 10 !important;
+}
+
+/* Styling for list card thumbnails */
+
+.gpap-list-card-thumbnail {
+  box-shadow: 0 1px 8px 0 rgba(0,0,0,.16);
+  padding:1rem;
```

```suggestion
.gpap-list-card-thumbnail {
  box-shadow: 0 1px 8px 0 rgba(0, 0, 0, 0.16);
  padding: 1rem;
```

These changes themselves look fine (except for the spacing), but as with #86 these changes shouldn't be in this file, they should be in the '.scss' files.